src/status_im2/contexts/chat/lightbox/text_sheet/view.cljs

```diff
@@ -13,13 +13,7 @@
     [status-im2.contexts.chat.lightbox.text-sheet.utils :as utils]
     [status-im2.contexts.chat.messages.content.text.view :as message-view]))
 
-(defn bar
-  [text-height]
-  (when (> text-height (* constants/line-height 2))
-    [rn/view {:style style/bar-container}
-     [rn/view {:style style/bar}]]))
-
-(defn text-sheet
+(defn- text-sheet
   [messages overlay-opacity overlay-z-index text-sheet-lock?]
```

Comment:
Just commenting for awareness, but it's up to you if you want to do anything about this. It wasn't introduced by your PR.

I was curious to see why we needed the collection `messages`, and it turns out only the first one is needed, and only `chat-id` and `content` keys are used. Reagent can perform significantly better the simpler the args we pass to components because the equality checks become cheaper, so it decides quicker if any given component should be recomputed. `message` is a data structure with a bunch of data.

It's not a measurable problem in isolation for the most part, but it's a good practice I think we should try to better follow. This practice is well documented too.

Reply:
Yeah it looked weird to me as well. Also with the lack of typing, I think we should be more explicit through the props we're passing. Hopefully with `malli` we'll think twice before passing unnecessary data, although I see this being an issue more with components outside quo2.